src/init/features/frameworks/repo.ts

```diff
@@ -30,21 +57,18 @@ function extractRepoSlugFromURI(remoteUri: string): string | undefined {
 
 /**
  * Generates a repository ID.
- * The relation is 1:* between Cloud Build Connection and Github Repositories.
+ * The relation is 1:* between Cloud Build Connection and GitHub Repositories.
  */
 function generateRepositoryId(remoteUri: string): string | undefined {
   return extractRepoSlugFromURI(remoteUri)?.replaceAll("/", "-");
 }
 
 /**
- * The 'frameworks-' is prefixed, to seperate the Cloud Build connections created from
- * Frameworks platforms with rest of manually created Cloud Build connections.
- *
- * The reason suffix 'location' is because of
- * 1:1 relation between location and Cloud Build connection.
+ * Generates connection id that matches speicifc id format recognized by all Firebase clients.
  */
-function generateConnectionId(location: string): string {
-  return `frameworks-${location}`;
+function generateConnectionId(): string {
+  const randomHash = Math.random().toString(36).slice(6);
```

Comment:
Good question. We actually want to have many firebase managed connections in a region since we may want to connect to different installation in the same region (e.g. repositories in Firebase org vs taeold account).